### PR TITLE
Fix Integration Tests - Closes #3266

### LINKED ISSUE #3266: Fix Integration Tests for feature/improve_transaction_efficiency
### Expected behavior

Integration Tests pass

### Actual behavior

Some Integration Tests are failing

### Steps to reproduce

Run integration tests

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

Some integration tests where still asserting unconfirmed state

### How did I fix it?

- By removing assertions against unconfirmed state
- Fixing other issues in integration tests

### How to test it?

- Integration tests should be green
- Unit Tests should be green
- Network Tests should be green
- Skipped functional tests will be fixed in #3152

### Review checklist

* The PR resolves #3266
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/test/mocha/functional/http/get/accounts/multisignatures.js [hunk 1]
```diff
@@ -65,14 +65,15 @@ describe('GET /api/accounts', () => {
 				return waitFor.confirmations([scenario.multiSigTransaction.id]);
 			});
 	});
-
+	// eslint-disable-next-line
```
Comment (lsilvs) [inline framework/test/mocha/functional/http/get/accounts/multisignatures.js]: Is this necessary?

File: framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js [hunk 2]
```diff
@@ -100,7 +100,12 @@ describe('integration test (type 1) - checking validated second signature regist
 				secondPassphrase: account.secondPassphrase,
 			});
 			localCommon.addTransaction(library, auxTransaction, err => {
-				expect(err).to.equal('Sender already has second signature enabled');
+				// expect(err).to.equal('Sender already has second signature enabled');
```
Comment (lsilvs) [inline framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js]: Shouldn't this be removed?

File: framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js [hunk 3]
```diff
@@ -120,7 +125,12 @@ describe('integration test (type 1) - checking validated second signature regist
 							true,
 							loadedTransaction => {
 								localCommon.addTransaction(library, loadedTransaction, err => {
-									expect(err).to.equal('Missing sender second signature');
+									// expect(err).to.equal('Missing sender second signature');
```
Comment (lsilvs) [inline framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js]: Shouldn't this be removed?

File: framework/test/mocha/functional/http/get/accounts/multisignatures.js (hunk 1 above)
Comment (pablitovicente) [reply framework/test/mocha/functional/http/get/accounts/multisignatures.js]: Removed

File: framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js (hunk 2 above)
Comment (pablitovicente) [reply framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js]: Removed

File: framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js (hunk 3 above)
Comment (pablitovicente) [reply framework/test/mocha/integration/transactions/1_second_signature/1_X_second_signature_validated.js]: Removed

**Pull request merged**